packages/react-reconciler/src/ReactFiberRootScheduler.js

```diff
@@ -315,6 +314,9 @@ function processRootScheduleInMicrotask() {
   if (!hasPendingCommitEffects()) {
     flushSyncWorkAcrossRoots_impl(syncTransitionLanes, false);
   }
+
+  // Reset Event Transition Lane so that we allocate a new one next time.
+  currentEventTransitionLane = NoLane;
```

Comment:
The comment above says the rendering work might throw, is that a problem?

Reply:
yea but it's not critical and it's only for a react internal bug